client/containers/Pub/PubDocument/PubDiscussions/Discussion/ThreadComment.tsx

```diff
@@ -134,16 +154,15 @@ const ThreadComment = (props: Props) => {
 								isSmall
 							/>
 						)}
-						<Editor
-							key={`${isEditing}-${threadCommentData.text}`}
-							isReadOnly={!isEditing}
-							initialContent={threadCommentData.content}
-							onChange={(editorChangeObject) => {
+						{renderText(
+							`${isEditing}-${threadCommentData.text}`,
+							true,
```

Comment:
I'm finding that I can't edit comments anymore and I'm wondering if you might want to pass `!isEditing` in here?